requirements/base.in

```diff
@@ -1,4 +1,4 @@
-Django >= 1.4, < 1.5
+Django
```

Comment:
Do we need a constraint or max version specified to prevent us from installing a major version bump?

Reply:
I saw one PR that seems to be a safe upgrade for django to me.
https://github.com/edx/xserver/pull/39

I'm still not sure about this as I have no idea about this repository.

Reply:
When removing the version constraint from `base.in`, you should instead pin the version in the `constraints.txt` file. This is the method that we use in most of the edX repos. Not adding the upper constraint on `Django` version will update it to the latest version whenever `django-nose` constraint is updated.